### ISSUE TITLE
OS X developer wanted

### ISSUE THREAD

**Issue by koppi** [cammill/cammill#21]
I did try to fix the OS X build in https://github.com/cammill/cammill/blob/master/.travis.yml
Here's a recent build log on OS X: https://travis-ci.org/cammill/cammill/jobs/66424530
I don't have an OS X machine, maybe someone with more time and an OS X machine can have a look into this.

**Comment by koppi** [comment]
@multigcs I just ran "make TARGET=OSX osx_app", the resulting dmg image file contains a png from one of your other projects, see screenshot:
![20150622-001](https://cloud.githubusercontent.com/assets/9458631/8289867/5ad53044-191f-11e5-95fe-1c90f2164334.png)

**Comment by multigcs** [comment]
right, i copy the generator-script vom my multigcs project, but we must only replace the dmg backgound image.

do you check the osx-app building ? (renamed: make TARGET=OSX package)
i can not test under osx :(

**Comment by koppi** [comment]
I don't have a local OS X machine, but travis-ci runs with an OS X node since a few days, see [.travis.yml](https://github.com/cammill/cammill/blob/master/.travis.yml) and build results at:
https://travis-ci.org/cammill/cammill

**Comment by multigcs** [comment]
![dmg-background](https://cloud.githubusercontent.com/assets/4227622/8292138/2ab1385e-192e-11e5-8ed2-73e5c4f4659b.png)

**Comment by koppi** [comment]
"make TARGET=OSX package" passes https://travis-ci.org/cammill/cammill/jobs/67895253

**Comment by koppi** [comment]
But:
```bash
cp: utils/cammill.icns: No such file or directory
```
see: https://travis-ci.org/cammill/cammill/jobs/67895253#L2737

**Comment by multigcs** [comment]
to genarate the cammill.icns you need 2 commands on osx:
 convert (imagemagick) and png2icns (???)
i can not generate this format under linux :( , but if we have this file, we can put it in git.

see : utils/osx-app.sh (## create/copy icons ##)

**Comment by koppi** [comment]
@multigcs https://github.com/cammill/cammill/commit/41561e8a259297a996ac75d69c87a1c1583167bd and https://github.com/cammill/cammill/commit/698b4f549c00e56802f1a9d59f9fba16dcda734f should do it.

**Comment by phatls** [comment]
Dear,
Your cammill project is very helpfull. If it's convinient, could you please help give me some guideline for setup enviroment to make plugin function or to custom some function?

Thanks

**Comment by koppi** [comment]
@phatls what are you trying to write code for?

**Comment by phatls** [comment]
@koppi: Translate into Vietnamese, some thing like that.

**Comment by koppi** [comment]
@phatls I just added vi.po to the project, but I ran into another issue, see: #45 hopefully @multigcs has a fix for this.

**Comment by phatls** [comment]
@koppi: I want not only Vietnamese but also add function to connect with camera, connect to cloud server, .. If you know, how to setting a project, pls help me.

Thanks

**Comment by koppi** [comment]
@phatls Can you describe in more detail, what we can help you with?
